src/Pie.js

```diff
@@ -57,37 +91,42 @@ const Pie = ({ sections, radius, innerRadius, backgroundColor,strokeCap}) => {
           stroke={backgroundColor}
           strokeWidth={width}
         />
-        <RingShape radius={radius} width={width} color={backgroundColor} />
+        <ArcShape radius={radius} width={width} color={backgroundColor} startAngle={0} arcAngle={360} />
         {sections.map((section, idx) => {
           const { percentage, color } = section;
 
-          const startAngle = startValue / 100 * 360
```

Comment:
`const { percentage, color, ...rest } = section;` and then `return <ArcShape {...rest} ... />` will resolve #18 #26

Reply:
Will do

Reply:
The ART components don't support many props like `onPress` and `testID`. We could wrap the pie in a `View` but I'm not sure what kind of regression that could cause. I will raise a separate PR for this

Reply:
OK, that's leave this for now